create_jenkins_job.py

```diff
@@ -216,14 +216,14 @@ def main(argv=None):
     job_config = expand_template('ci_launcher_job.xml.em', job_data)
     configure_job(jenkins, 'ci_launcher', job_config, **jenkins_kwargs)
 
-
     # Run the turtlebot job on Linux only for now.
     os_name = 'linux'
     turtlebot_job_data = dict(data)
     turtlebot_job_data['os_name'] = os_name
     turtlebot_job_data.update(os_configs[os_name])
     turtlebot_job_data['turtlebot_demo'] = True
-    turtlebot_job_data['default_repos_url'] = 'https://raw.githubusercontent.com/ros2/turtlebot2_demo/master/turtlebot2_demo.repos'
+    # Will use a turtlebot2_demo repos file when one is available.
+    # turtlebot_job_data['default_repos_url'] = 'https://raw.githubusercontent.com/ros2/turtlebot2_demo/master/turtlebot2_demo.repos'
```

Comment:
This is totally fine, but just for your understanding of how the things inter-relate: even with the `default_repos_url` set to a url that doesn't exist yet, it is just the default. The url can be specified to be something other than the default using the `CI_ROS2_REPOS_URL` on jenkins when doing "Build with Parameters"